mailtrap/client.py

```diff
@@ -72,6 +73,14 @@ def contacts_api(self) -> ContactsBaseApi:
             client=HttpClient(host=GENERAL_HOST, headers=self.headers),
         )
 
+    @property
+    def suppressions_api(self) -> SuppressionsBaseApi:
```

Comment:
Based on the API dosc, the suppressions_api should be part of the sending API :thinking:. On the other hand, the NodeJS SDK uses the same design (suppressions api is a separate resource). So I guess we are good here, let's better be aligned with other SDKs